code/dataobjects/DynamicList.php

```diff
@@ -36,6 +36,43 @@ public function getCMSFields() {
 			$fields->addFieldToTab('Root.Items', new GridField('Items', 'Dynamic List Items', $this->Items(), $conf));
 		}
 
+		// Make sure the draft records are being looked at.
+
+		$stage = Versioned::current_stage();
+		Versioned::reading_stage('Stage');
+		$used = EditableFormField::get()->filter(array(
```

Comment:
Ah yes, sorry, I should have picked up on that. It was originally an extension to begin with, so I will pull that back out.